test/run_test.py

```diff
@@ -1017,7 +1017,9 @@ def export_S3_test_times(test_times_filename: str, test_times: Dict[str, float])
         print(f'Overwriting existent file: {test_times_filename}')
     with open(test_times_filename, 'w+') as file:
         job_times_json = get_job_times_json(test_times)
-        json.dump(job_times_json, file)
+        json.dump(job_times_json, file, indent='    ', separators=(',', ': '))
+    with open(test_times_filename, 'a') as file:
```

Comment:
Why do you close the file and then reopen it again to append a newline?
```suggestion
```

Reply:
I was under the impression that w+ and file.write() would overwrite the whole thing again, but now I realize I misread the docs :c